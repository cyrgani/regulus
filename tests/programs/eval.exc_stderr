SyntaxError: missing or invalid tokens for s_step

DivideByZeroError: attempted to divide by zero
at <stl:operators>:59:26
at <stl:operators>:57:11
at <stl:operators>:53:19
at <file>:1:4
at <file>:1:2

NameError: No variable named `aaaaa` found!

