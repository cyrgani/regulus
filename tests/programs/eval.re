assert_eq(eval("+(2,2)"), 4),
assert_eq(catch(eval("(")), "SyntaxError: Nonequal amount of '(' and ')': 1 vs. 0"),
=(x, "_("),
=(y, strconcat(x, "/(1, 0))")),
assert_eq(catch(eval(y)), "OverflowError: overflow occured during /!"),

eval("=(aaaaa, 2)"),
assert_eq(catch(aaaaa), "NameError: No variable named `aaaaa` found!"),