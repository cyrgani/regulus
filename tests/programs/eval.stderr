SyntaxError: missing or invalid tokens for s_step
DivideByZeroError: attempted to divide by zero
at <stl:operators>:55:26
at <stl:operators>:53:11
at <stl:operators>:50:19
at <file>:1:4
at <file>:1:2
NameError: No variable named `aaaaa` found!
at programs/eval.re:8:23
at programs/eval.re:8:22
